pages/builds/parallelizing_builds.md.erb

```diff
@@ -51,29 +51,28 @@ BUILDKITE_PARALLEL_JOB=0
 
-The `BUILDKITE_PARALLEL_JOB_COUNT` environment variable is the total number of jobs.
+The `BUILDKITE_PARALLEL_JOB_COUNT` environment variable stores the total number of jobs created from this step for this build.
 
 You can use these two environment variables to divide your application’s tests between the different jobs.
 
 ## Libraries
 
-The following libraries have built-in support for the `BUILDKITE_PARALLEL_*` environment variables:
+The [Knapsack](https://github.com/ArturT/knapsack) ruby gem has built-in support for the `BUILDKITE_PARALLEL_JOB` and `BUILDKITE_PARALLEL_JOB_COUNT` environment variables:
 
-* [Knapsack](https://github.com/ArturT/knapsack)
-<br>A ruby gem for that will automatically divide your tests between parallel jobs, as well as making sure each job runs in comparable time. Supports RSpec, Cucumber and Minitest.
+Knapsack is a ruby gem for automatically dividing your tests between parallel jobs, as well as making sure each job runs in comparable time. It supports RSpec, Cucumber, and Minitest.
 
 ## Isolated jobs
 
 You can safely run multiple agents on a single machine, though you’ll still need to ensure your application’s code is able to be run in parallel. One convenient way of doing this is by using the agent’s [built in Docker and Docker Compose support](docker-containerized-builds) which will run each job in one or more completely isolated containers.
 
 ## Auto-scaling your build agents
 
-We've provided a number of APIs to allow you to automatically scale your build agents based on build queue.
+We provide a number of APIs for automatically scaling agents based on you build queue.
 
-* Using the [Pipelines API](/docs/api/pipelines) you’re able to fetch the `scheduled_jobs_count`, `waiting_jobs_count` and `running_jobs_count` to get an overview of the number of jobs waiting to be run. Based on this you can spin up new agent machines to process the jobs waiting to be run.
-* Using [agent prioritization](/docs/agent/prioritization) you’ll able to define a core set up agents that are assigned work first, and then lower priority agents which you may spin up and spin down on demand.
-* Using the [Agents API](/docs/api/agents) you’re able to fetch your agent’s `job` and `last_job_finished_at`, which allow you to see if they have been idle. You can then remotely shut down an agent process using the [agent stop API](/docs/api/agents#stop-an-agent) (passing `"force": false` to ensure it won’t cancel any running job).
+* Using the [Pipelines API](/docs/api/pipelines) you’re able to fetch the `scheduled_jobs_count`, `waiting_jobs_count` and `running_jobs_count` to get an overview of the number of jobs waiting to be run. Keeping track of this value, you can spin new agent machines up or down to accomodate the job load.
+* Using [agent prioritization](/docs/agent/prioritization) you’ll able to define which agents are assigned work first, as well as lower priority agents which you may spin up and down on demand.
```

Comment:
If you hit the "Stop" button on the agent page, they shut themselves down. But in normal day to day work, the agents will stay connected until they're told to shutdown.